doc/kubernetes/modules/ROOT/pages/util/custom-image-for-keycloak.adoc

```diff
@@ -0,0 +1,64 @@
+= Using a custom Keycloak image for deployment in Kubernetes
+:navtitle: Using a custom Keycloak image
+:description: Use a custom image an external registry or a locally built image and run this in Kubernetes.
+
+{description}
+A running version of xref:installation.adoc[Keycloak within Minikube] is a prerequisite.
+
+== Overview
+
+Depending on the operator being used, it will pull a default image from the Keycloak's Quay.io registry.
+When running the nightly operator of Keycloak, it uses the nightly build of Keycloak.
+A specific version of the operator like, for example, 20.0.0, will use the version 20.0.0 of Keycloak.
+
+== Use a custom image available in a remote registry
+
+To change this, edit the environment file `.env` to contain the following:
+
+.Example entry in the `.env` file
+----
+KC_KEYCLOAK_IMAGE=quay.io/keycloak/keycloak:20.0.1
+----
+
+== Building a custom Keycloak image and using it
+
+. Check out https://github.com/keycloak/keycloak[Keycloak's Git repository].
+. Build using `mvn clean install -DskipTests` to create a `keycloak-999-SNAPSHOT.tar.gz` in folder `/quarkus/container`.
```

Comment:
```suggestion
. Build using `mvn clean install -DskipTests` to create a `keycloak-999-SNAPSHOT.tar.gz` in folder `/quarkus/dist/target`.
```

Reply:
I think we should alter the directory as the tar ball is created under the `quarkus/dist/target` instead of `quarkus/container`